source/javascript/src/daily.js

```diff
@@ -331,3 +340,60 @@ const custZoom = document.getElementById('button1')
 custZoom.addEventListener('click', function () {
   zoom.click()
 })
+
+/**
+ * Business logic for tommorow Button
+ * @author Brett Herbst <bherbst@ucsd.edu>
+ * Modeled after index.js Rapid Log button
+ */
+tmButton.addEventListener('click', () => {
+  const wrapper = new IndexedDBWrapper('experimentalDB', 1)
+  wrapper.transaction((event) => {
+    const db = event.target.result
+
+    const transaction = db.transaction(['currentLogStore'], 'readwrite')
+
+    const store = transaction.objectStore('currentLogStore')
+
+    const req = store.openCursor()
+    // Fires when cursor is successfully opened.
+    req.onsuccess = function (e) {
+      const cursor = e.target.result
+      if (cursor) {
+        // Iterates current_log forward 1 day
+        cursor.value.current_log = (parseInt(cursor.value.current_log) + 86400000).toString()
```

Comment:
`+86400000` is probably not a good practice. The user can't click the chevron more than once. Using current_log plus an offset could potentially be a good practice if we can keep current_log up to date, i.e. always pointing at the current log the user is looking at. @NoahT I think we need to think deeper into how to integrate the router.js. @brettherbst I am going to merge this pr, but please make another issue handling this bug

Reply:
Once we transition to using `Router.js` for all navigation will we replace the `current_log` property by putting the UNIX timestamp of the current log as a query parameter in the URL?

Reply:
@yuzilyu good point, this will be a temp fix hopefully using router.js I can make it a more permanent fix later

Reply:
@will-chung correct, we're going to remove `current_log` and `current_collection` since it's a solution that's prone to errors (the biggest one being if a user wants to have multiple tabs of our bullet journal open at once)

Reply:
@yuzilyu good point. It depends on what information we want to encode into the URL for the different points of navigation to each page (i.e. for the collection-edit page, we only need the URL-encoding of the name, but for daily-log, we can maybe pass in a timestamp or something along those lines).